src/wallet/walletdb.cpp

```diff
@@ -902,8 +909,7 @@ bool WalletBatch::RecoverKeysOnlyFilter(void *callbackData, CDataStream ssKey, C
     {
```

Comment:
in commit 6d05dff02be9e113ac6f835:

The now unused `CWalletScanState dummyWss;` a few lines up should now be removed.

Reply:
Removed